app/models/user.rb

```diff
@@ -83,4 +86,12 @@ def email_from_symphony
       SymphonyUserNameRequest.new(libid: library_id).email
     end if library_id_user?
   end
+
+  private
+
+  def notify_honeybadger_of_unknown_webauth_email!
+    Honeybadger.notify(
+      "Webauth user record being saved without a valid email address. Using #{webauth}@stanford.edu instead."
+    ) if new_record?
```

Comment:
Is it possible for this method to be called when retrieving and old user, and that we're not actually trying to save the user at this point? Just wondering if the Honeybadger message needs to be massaged a little here.